tests/checkpoint/test_checkpoint_result_format.py

```diff
@@ -2851,76 +2766,6 @@ def test_pandas_result_format_in_checkpoint_named_index_two_index_column_not_set
     assert first_result_partial_list == [("three"), ("four"), ("five")]
 
 
-@pytest.mark.integration
```

Comment:
@Kilo59 Similarly -- please ask @Shinnnyshinshin to review carefully.  Thank you.